static/js/containers/pages/CheckoutPage.js

```diff
@@ -104,19 +106,21 @@ export class CheckoutPage extends React.Component<Props, State> {
     const { productId } = this.getQueryParams()
     if (!productId) {
```

Comment:
`this.setState({ isLoading: true })` right when you come into this method

Reply:
isLoading state comes **true** as a default value in this component.

Reply:
I guess that's right